_CoqProject

```diff
@@ -0,0 +1,10 @@
+-R VST-2.3/compcert/lib compcert.lib
+-R VST-2.3/compcert/flocq compcert.flocq
+-R VST-2.3/compcert/x86_32 compcert.x86_32
+-R VST-2.3/compcert/common compcert.common
+-R VST-2.3/sha sha
+-R VST-2.3/msl VST.msl
+-R VST-2.3/veric VST.veric
+-R VST-2.3/floyd VST.floyd
+-R Coq/Util Util
```

Comment:
I think we should prefer to use `-Q` instead of `-R` for these flags, which is what I use in `Simplicity.Coq.nix`.